src/lib/containers/UserCardSmall.tsx

```diff
@@ -107,6 +107,15 @@ export const UserCardSmall: React.FunctionComponent<UserCardSmallProps> = ({
     <></>
   )
 
+  const fullName = showFullName ? (
+    <span className={'user-fullname'}>
+      &nbsp;
+      {`${userProfile.firstName ?? ''}`}&nbsp;{`${userProfile.lastName ?? ''}`}
```

Comment:
If firstName or lastName is undefined, use the empty string. Fixes "@username undefined undefined"